ceph/rados/core_workflows.py

```diff
@@ -3399,3 +3399,66 @@ def log_cluster_health(self):
         health_detail, _ = self.node.shell(args=["ceph health detail"])
         log.info(f"\n****\n Cluster health detail: \n {health_detail} \n ****")
         return health_detail
+
+    def create_ecpool_inconsistent_obj(
+        self, pool_object, client_node, pool_name, no_of_objects
+    ):
+        """
+        The method converts the object into inconsistent object in EC pool
+        The logic implemented in the code is-
+        1. Get the target osd and pg_id
+        2. Get the object details by using the ceph-objectstore-tool
+        3. Remove the “hinfo_key” of the object
+        4. perform scrub
+        Args:
+            pool_object: pool object
+            client_node: client object
+            pool_name: pool name
+            no_of_objects: no of objcts to convert in to inconsistent objects
+        Returns: After converting the object in to inconsistent,method returns the pg id
+                 If it fail returns None
+        """
+        for obj_num in range(no_of_objects):
+            object_name = f"object_{obj_num}"
+            cmd_create_obj = f"rados -p {pool_name} put {object_name} /etc/group"
+            client_node.exec_command(cmd=cmd_create_obj, sudo=True)
+
+            osd_map_output = self.get_osd_map(pool=pool_name, obj=object_name)
+            log.debug(
+                f"\nThe acting set details for the object : {object_name}"
+                f" on pool : {pool_name} is {osd_map_output}\n"
+            )
+            target_osd = osd_map_output["acting_primary"]
+
+            log.info(f"The object stored in the target osd number-{target_osd}")
+            pg_id = osd_map_output["pgid"]
+            log.info(f"The object {object_name} is created in the pg-{pg_id}")
+            # stoping the OSD
+            if not self.change_osd_state(action="stop", target=target_osd):
```

Comment:
Let's corrupt all the objects in single start/stop of OSD.

This would make test more robust and consumes less time.

Reply:
Changed the logic, while creating inconsistent objects with stopping and starting  one OSD.